config/dev1/sys.config

```diff
@@ -50,7 +50,7 @@
              {handlers, [
                {lager_file_backend, [
                   {file, "log/epoch_mining.log"},
-                  {level, info}
+                  {level, debug}
```

Comment:
Isn't `info` a better hard-coded default? If we want `debug` for the testnet, we should really have a user config to set the loglevel for the mining log. But for now, I guess this is ok.